### DISCUSSION TITLE
Unable to Install Fresh SecOnion Fails at 95%

### DISCUSSION THREAD

**Discussion by mattcampbell15** [Security-Onion-Solutions/securityonion#5102] [Unsupported Versions] [answered]
Hi,

 I am attempting to install Security Onion in a VM, I attempted multiple installs and fails at 95%. 

ISO using
securityonion-2.3.61-MSEARCH.iso
Install Type Standalone (No osquery or WAZU)

Current specs of the VM
8Cores
12GB of ram
120GB for the OS
1tb for the NSM storage
Dual Nic ( i double check the correct Management NIc and monitor port)

When Running the following command sudo salt-call state.highstate

[secroot@seconion ~]$ sudo salt-call state.highstate
local:
    Data failed to compile:
----------
    The function "state.highstate" is running as PID 3143 and was started at 2021, Aug 07 23:01:24.863152 with jid 20210807230124863152

Running so-status

    so-aptcacherng ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-cortex ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-curator --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-dockerregistry -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-elastalert ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-elasticsearch ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ ERROR ]
    so-filebeat -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-grafana --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-idstools -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-influxdb -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-kibana ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-kratos ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-logstash -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-mysql ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-nginx ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-playbook -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-redis ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-sensoroni ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-soc ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-soctopus -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-steno ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-backend ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-coordinator --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-filestream ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-frontend ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-strelka-gatekeeper ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-manager ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-suricata -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-telegraf -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-thehive --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-thehive-es ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-zeek ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]

It appears Elastactic Search can't start, I wonder if its a DNS issue when im setting the DNS suffix during the setup which is starfleet.int for me
When i query the setup logs i see the below

't connect to any of the configured Elasticsearch hosts. Errors: [error connecting to Elasticsearch at https://seconion:9200: Get "https://seconion:9200": dial tcp 172.16.0.50:9200: connect: connection refused]
                  Exiting: couldn't connect to any of the configured Elasticsearch hosts. Errors: [error connecting to Elasticsearch at https://seconion:9200: Get "https://seconion:9200": dial tcp 172.16.0.50:9200: connect: connection refused]
                  Exiting: couldn't connect to any of the configured Elasticsearch hosts. Errors: [error connecting to Elasticsearch at https://seconion:9200: Get "https://seconion:9200": dial tcp 172.16.0.50:9200: connect: connection refused]
                  Exiting: couldn't connect to any of the configured Elasticsearch hosts. Errors: [error connecting to Elasticsearch at https://seconion:9200: Get "https://seconion:9200": dial tcp 172.16.0.50:9200: connect: connection refused]
                  Exiting: couldn't connect to any of the configured Elasticsearch hosts. Errors: [error connecting to Elasticsearch at https://seconion:9200: Get "https://seconion:9200": dial tcp 172.16.0.50:9200: connect: connection refused]
                  Exiting: couldn't connect to any of the configured Elasticsearch hosts. Errors: [error connecting to Elasticsearch at https://seconion:9200: Get "https://seconion:9200": dial tcp 172.16.0.50:9200: connect: connection refused]
                  Exiting: couldn't connect to any of the configured Elasticsearch hosts. Errors: [error connecting to Elasticsearch at

**Comment by mattcampbell15** [comment]
![image](https://user-images.githubusercontent.com/34895556/128616185-c1e3047b-7b14-4655-8eaf-86887116ed78.png)
/etc/hosts file

**Comment by gcsrellis** [comment, marked as answer]
12GB Ram is the minimum requirement for EVAL mode.  You may not have enough for your application.

**Reply by mattcampbell15** [reply]
apparently i can't read and i miss read the guide, i up it to 16gb of ram and it worked.

Thank you. Sorry for wasting anyone's time who read this
